params/protocol_params.go

```diff
@@ -28,7 +28,7 @@ const (
 	GenesisGasLimit      uint64 = 4712388 // Gas limit of the Genesis block.
 	XDCGenesisGasLimit   uint64 = 84000000
 
-	MaximumExtraDataSize  uint64 = 32    // Maximum size extra data may be after Genesis.
+	MaximumExtraDataSize  uint64 = 65    // Maximum size extra data may be after Genesis.
```

Comment:
it seems this `MaximumExtraDataSize` is not used in our codebase. otherwise our extra already exceeds it

Reply:
yes, this is in ethash consensus, just change this for passing test, if I remove this, more and more code is needed to be clean up.My target is removing v1 code and fix all the tests.